easypy/collections.py

```diff
@@ -553,27 +560,28 @@ def index(self, obj):
             if lookup_uid == uid:
                 return i
 
-    def get_next(self, obj):
-        uid = self._get_object_uid(obj)
-        if uid not in self._objects:
-            raise ObjectNotFound(self, (), dict(key=uid),)
-        next_uid = self._objects._OrderedDict__map[uid].next
-        try:
-            key = next_uid.key
-        except AttributeError:
-            key = next_uid.next.key
-        return self._objects[key]
-
-    def get_prev(self, obj):
-        uid = self._get_object_uid(obj)
-        if uid not in self._objects:
-            raise ObjectNotFound(self, (), dict(key=uid),)
-        prev_uid = self._objects._OrderedDict__map[uid].prev
-        try:
-            key = prev_uid.key
-        except AttributeError:
-            key = prev_uid.prev.key
-        return self._objects[key]
+    if SUPPORT_GET_SIBLING:
```

Comment:
We should have an `else:` with O(n) implementations, just to avoid this weird breakage.

Reply:
will do, but I'm actually thinking to remove get_prev/get_next, I don't think they ever became very popular...